### PR TITLE
More.static

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

**Comment by stuertz** [comment]
@conda-forge/core It is ready for merge, when my other pending PR containing the dependencies are merged ;-)

File: recipes/more.static/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  # Python imports
+  imports:
+    - more
+    - more.static
+
+about:
+  home: https://github.com/morepath/more.static
+  license: BSD
```
Comment (ocefpaf) [inline recipes/more.static/meta.yaml]: Needs the license version.

**Comment by stuertz** [comment]
@ocefpaf Hi, this is ready for merge. Dependencies are all met. You may restart circleci. Thanks.

File: recipes/more.static/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - morepath >=0.16
+    - bowerstatic >=0.8
+
+  run:
+    - python
+    - morepath >=0.16
+    - bowerstatic >=0.8
+
+test:
+  # Python imports
```
Comment (ocefpaf) [inline recipes/more.static/meta.yaml]: Please remove this comment. (It can be done in the feedstock.)

**Pull request merged**